### DISCUSSION TITLE
一些问题

### DISCUSSION THREAD

**Discussion by Makitoid** [ClassIsland/ClassIsland#486] [Q&A] [answered]
*我不知道这是不是bug，先发个discussion问问*
1.在属性里把classisland设置为默认以管理员模式运行，就不能开机自启了，任务管理器的启动中有classisland，设置里也打开了自启 
   动，就是开机不启动
2.下午第一节课我明明设置了即将上课提醒，就是不显示，如图是我的课表设置
![tmp_30956d42-71f1-48b5-8cec-936c2e2ab7bd](https://github.com/user-attachments/assets/19ef9512-c95c-4248-bde0-1964c0287c6e)

**Comment by DryIce-cc** [comment, marked as answer]
1. 参见 #319 中的解决方法
2. 这是一个 Bug。在稳定通道下测试，这里 `LessonsService.OnClassLeftTime` 提供的时间为负。@HelloWRC

**Reply by Makitoid** [reply]
> 2. 这是一个 Bug
@DryIce-cc  测试版1.5.1.0试了，没这个bug

**Comment by Makitoid** [comment]
对于1，能不能在程序里加个`以管理员身份运行`的功能？毕竟弄个计划任务比较麻烦
对于2，需要报告这个bug吗？（是不是可以通过在午休设一个超长的课间然后隐藏暂时解决这个问题？）
谢谢开发者大大了
